client/landing/stepper/hooks/use-is-domain-code-valid.ts

```diff
@@ -61,6 +61,7 @@ export function useIsDomainCodeValid( pair: DomainCodePair, queryOptions = {} )
 						domain: pair.domain,
 						status: availability.status,
 						unlocked: false,
+						transferrability: availability?.transferrability ?? null,
```

Comment:
```suggestion
						transferrability: availability.transferrability,
```
I don't think the optional chain and null coalescing operators are needed here because `availability` is guaranteed (I think 🙂 ) to be an object at this point, otherwise, if an error occurred we'd be in the `catch` block